site/docs/plugins/autoquote.md

```diff
@@ -0,0 +1,56 @@
+# Always replying to messages
+
+Sometimes, especially for bots that are meant to be used in groups, it is necessary to always send messages as a reply (or quote) to the message that started the interaction. Usually, the way to do that is to mannualy add `reply_to_message_id` to the params of the mathod that sends the message (`sendText` / `reply`, `sendPhoto` / `replyWithPhoto`). However, if you're doing this for every message, it can get a bit messy and tiring.
```

Comment:
```suggestion
Sometimes, especially for bots that are meant to be used in groups, it is necessary to always send messages as a reply (or quote) to the message that started the interaction.
Usually, the way to do that is to mannualy add `reply_to_message_id` to the params of the mathod that sends the message (`sendText` / `reply`, `sendPhoto` / `replyWithPhoto`).
However, if you're doing this for every message, it can get a bit messy and tiring.
```